src/OrbitGl/CaptureViewElement.cpp

```diff
@@ -65,8 +65,7 @@ CaptureViewElement::EventResult CaptureViewElement::OnMouseMove(const Vec2& mous
     std::ignore = OnMouseEnter();
   }
 
-  mouse_pos_cur_ = mouse_pos;
-  // Ignoring the event, so mouse position is updated for its ancestors as well.
+  // Ignoring the event, so mouse move is executed for its ancestors as well.
```

Comment:
NIT: It would be better to use the imperative. 

"Ignore this event and let it be handled by its ancestors."